openqabot/types/baseconf.py

```diff
@@ -1,17 +1,17 @@
 # Copyright SUSE LLC
 # SPDX-License-Identifier: MIT
-from abc import ABCMeta, abstractmethod, abstractstaticmethod
```

Comment:
Why ?

Reply:
Because I need to test `is_embargoed` method , for this I need to create instance of this class see test code

Reply:
test can be on Any other class which inherits  from BaseConf ......

Reply:
ok will do that

Reply:
done